locales/zh_CN/LC_MESSAGES/vector.po

```diff
@@ -186,58 +170,52 @@ msgstr ""
 
 #: ../../vector.rst:104
 msgid "Methods"
-msgstr ""
+msgstr "相关方法"
 
 #: ../../vector.rst:108 ../../vector.rst:127 ../../vector.rst:128
 #: ../../vector.rst:155 ../../vector.rst:156 ../../vector.rst:173
 msgid "(Vector)"
-msgstr ""
+msgstr "(Vector)向量"
 
 #: ../../vector.rst:109
-msgid ""
-"(optional, scalar) a safe-guard value for ``sqrt``, usually 0. See the "
-"note below."
-msgstr ""
+msgid "(optional, scalar) a safe-guard value for ``sqrt``, usually 0. See the note below."
+msgstr "(optional, scalar) sqrt的安全保护值，通常为0. 详见下面注解。"
 
 #: ../../vector.rst
 msgid "Returns"
 msgstr ""
 
 #: ../../vector.rst:110
 msgid "(scalar) the magnitude / length / norm of vector"
-msgstr ""
+msgstr "(scalar) 向量的大小、长度、范数"
 
 #: ../../vector.rst:112
 msgid "For example, ::"
-msgstr ""
+msgstr "E.g., ::"
 
 #: ../../vector.rst:115
-msgid ""
-"a = ti.Vector([3, 4])\n"
+msgid "a = ti.Vector([3, 4])\n"
 "a.norm() # sqrt(3*3 + 4*4 + 0) = 5"
 msgstr ""
 
 #: ../../vector.rst:118
 msgid "``a.norm(eps)`` is equivalent to ``ti.sqrt(a.dot(a) + eps)``"
-msgstr ""
+msgstr "``a.norm(eps)`` 相当于 ``ti.sqrt(a.dot(a) + eps)``"
 
 #: ../../vector.rst:121
-msgid ""
-"Set ``eps = 1e-5`` for example, to safe guard the operator's gradient on "
-"zero vectors during differentiable programming."
-msgstr ""
+msgid "Set ``eps = 1e-5`` for example, to safe guard the operator's gradient on zero vectors during differentiable programming."
+msgstr "可以通过设置 ``eps = 1e-5`` ,对可微分编程中零向量上的梯度值计算进行防护。"
 
 #: ../../vector.rst:129
 msgid "(scalar) the dot (inner) product of ``a`` and ``b``"
-msgstr ""
+msgstr "(scalar) ``a`` 和 ``b`` 之间点乘（内积）的结果"
 
 #: ../../vector.rst:131 ../../vector.rst:159 ../../vector.rst:177
 msgid "E.g., ::"
-msgstr ""
+msgstr "E.g., ::"
```

Comment:
Not translated? Maybe some cn reader can't understand what `e.g.` mean?

Reply:
Sorry for my neglect of double checking for the content.